src/main/java/de/tum/in/www1/artemis/repository/StudentParticipationRepository.java

```diff
@@ -117,26 +117,45 @@
     Optional<StudentParticipation> findByExerciseIdAndTeamIdWithLatestResult(@Param("exerciseId") Long exerciseId, @Param("teamId") Long teamId);
 
     /**
+     * TODO SE, NR: maybe re-add
```

Comment:
This can be removed now right?

Reply:
For the moment we will keep it, will be removed in a follow up shortly. Probably next week.